### PR TITLE
Delet license

### PR DESCRIPTION
**Pull request by Galbert01**
Rm

# Thank you for your interest in improving GitHub's site policies!
new Please follow this workflow to have your edits considered:
1. Open a pull request directly in the [GitHub Docs](https://github.com/github/docs) repo.
2. Open [an issue in this site-policy repo](https://github.com/github/site-policy/issues/new) describing your change and link it to your PR.
3. We will then consider your change and get back to you. Thank you!

### CODE REVIEW

**Comment by Galbert01** [comment]
Rm

**Review by EnsDeLiz — APPROVED** [review]
sudo1393@gmail.com

**Review by EnsDeLiz — APPROVED** [review]
sudo1393@gmail.com

File: LICENSE.md [hunk 1]
```diff
@@ -1,116 +0,0 @@
-CC0 1.0 Universal
-
```
Comment (EnsDeLiz) [inline LICENSE.md:2]: <<sudo1393@gmail.com>>

Comment (EnsDeLiz) [reply LICENSE.md:2]: Ok

**Review by EnsDeLiz — APPROVED** [review]

**Pull request closed without merging**